best-practices/native/react-native/training/README.md

```diff
@@ -0,0 +1,147 @@
+# Table of contents
```

Comment:
I believe this page should be added to a parent page such as: https://github.com/Shift3/standards-and-practices/blob/main/best-practices/client-side/framework/react/README or similar.